manifest.json

```
@@ -1,7 +1,7 @@
 {
   "vendor": "vtex",
   "name": "store-graphql",
-  "version": "2.171.0",
+  "version": "2.171.1-beta",
```

```suggestion
  "version": "2.171.0",
```

You can simply keep it as a patch because when CI/CD runs it will update the version with vtex release